### PR TITLE
Added Magento products and categories imports

### PR DESCRIPTION
**Pull request by kevinmeijer97**

### CODE REVIEW

File: resources/content/collections/categories.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+layout: layout
+revisions: false
+slugs: false
+sort_dir: asc
+date_behavior:
+  past: public
+  future: private
+structure:
+  root: false
+  slugs: true
```
Comment (BobWez98) [inline resources/content/collections/categories.yaml:13]: above there's `slugs: false`, what's the diference?

File: src/Actions/Categories/CreateCategories.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            return new Collection();
+        }
+
+        return $categories->filter(fn ($category) => !blank($category->name) && !blank($category->url_path) && !blank($category->entity_id))->map(fn ($category) => [
+            'magento_entity_id' => $category->entity_id,
+            'magento_parent_id' => $category->parent_id ?? '',
+            'title' => $category->name,
+            'slug' => Str::replace('/', '-', $category->url_path),
+            'path' => $category->url_path ?? '',
+        ])->each(function ($category) use ($storeCode) {
```
Comment (BobWez98) [inline src/Actions/Categories/CreateCategories.php:23]: if you use a shorhand function, you can remove the use.

File: resources/content/collections/categories.yaml (hunk 1 above)
Comment (kevinmeijer97) [reply resources/content/collections/categories.yaml:13]: This option shows the slug in the tree view in the CP. This was not necessary for the default view.

File: src/Actions/Categories/CreateCategories.php (hunk 2 above)
Comment (kevinmeijer97) [reply src/Actions/Categories/CreateCategories.php:23]: Gotcha, gonna check the rest as well!

File: src/Http/Controllers/StatamicRewriteController.php [hunk 3]
```diff
@@ -5,17 +5,25 @@
...
 
 class StatamicRewriteController
 {
     public function __invoke(Request $request)
     {
+        $storeCode = config()->get('rapidez.store_code');
+
         $entry = Entry::query()
             ->where('collection', 'pages')
+            ->where('locale', $storeCode)
```
[the remark this answers is not in the document]
Comment (kevinmeijer97) [reply src/Http/Controllers/StatamicRewriteController.php:18]: Locale is not actually the locale on an entry, instead it's the site. A bit confusing..

**Pull request merged**